src/ucp/core/ucp_worker.c

```diff
@@ -1882,8 +1897,10 @@ ucs_status_t ucp_worker_get_ep_config(ucp_worker_h worker,
                                         UCP_PROTO_FLAG_AM_SHORT, key->am_lane,
                                         &ep_config->am_u.max_eager_short);
     } else {
-        ucs_info("%s", ucp_worker_print_used_tls(key, context, ep_cfg_index,
-                                                 &strb));
+        str = ucp_worker_print_used_tls(key, context, ep_cfg_index, &strb);
```

Comment:
maybe ucp_worker_print_used_tls could call ucs_info() inside of it (and return void) ?

Reply:
done